lib/lndclient/LndClient.ts

```diff
@@ -575,6 +591,7 @@ class LndClient extends SwapClient {
   };
 
   public sendSmallestAmount = async (rHash: string, destination: string): Promise<string> => {
+    // TODO: as of lnd 0.12.0, this won't work as PaymentAddr will be required and the only way to specify it will be with the pay req
```

Comment:
We should update this one too asap. Maybe using keysend could be an alternative? Will LND 0.12 also break keysends?

Reply:
I'm not sure about keysends, but since this code isn't being used right now it's not urgent. This whole method could be a todo stub for the time being since we're not using sanity swaps which is what this is for.